extensions/dacpac/src/test/testDacFxService.ts

```diff
@@ -139,7 +139,22 @@ export class DacFxTestService implements mssql.IDacFxService {
 				dropObjectsNotInSource: false,
 				ignoreColumnOrder: false,
 				doNotDropObjectTypes: [],
-				excludeObjectTypes: []
+				excludeObjectTypes: [],
```

Comment:
These changes need to be updated!

Reply:
Its done on other PR https://github.com/microsoft/azuredatastudio/pull/18050/files#diff-4b1525d27fb442502c34c4135a85b3c68f5ced8cb4ad27d0ef1d7fa5fe5050f0